### PR TITLE
Refactor Gradle Plugin things

### PR DESCRIPTION
**Pull request by brfrn169**
Please take a look!

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -48,6 +48,11 @@ subprojects {
         errorproneJavacVersion = '9+181-r4173-1'
         gsonVersion = '2.9.0'
         log4jVersion = '2.17.1'
+        spotbugsPluginVersion  = '4.7.1'
+        errorpronePluginVersion  = '2.0.2'
+        protobufPluginVersion = '0.8.17'
+        shadowPluginVersion = '7.1.2'
+        dockerPluginVersion = '0.25.0'
```
Comment (brfrn169) [inline build.gradle:55]: Made the Gradle plugin versions variables.

File: schema-loader/build.gradle [hunk 2]
```diff
@@ -141,7 +141,7 @@ docker {
 
 task dockerfileLint(type: Exec) {
     description 'Lint the Dockerfile'
-    commandLine "${project.rootDir}/ci/scripts/dockerfile_lint.sh"
+    commandLine "${project.rootDir}/ci/dockerfile_lint.sh"
```
Comment (brfrn169) [inline schema-loader/build.gradle:144]: Also, moved `dockerfile_lint.sh`.

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Review by Torch3333 — APPROVED** [review]
LGTM, thank you!

**Pull request merged**